charts/opentelemetry-operator/values.yaml

```
@@ -62,6 +62,11 @@ admissionWebhooks:
       # kind:
       # name:
 
+## Create the provided ClusterRoles and ClusterRoleBindings
+##
+rbac:
+  create: true
```

```suggestion
  enabled: true
```
The charts usually use `enabled` as the field to turn things on and off